### PR TITLE
Merge `use_repo` buildifier fixups into a single command

### PR DESCRIPTION
**Pull request by fmeum**
Users only have to copy&paste a single command this way.

### CODE REVIEW

**Comment by fmeum** [comment]
Thanks @alexeagle for making me aware of this Buildozer syntax.

File: src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionMetadata.java [hunk 1]
```diff
@@ -354,8 +354,7 @@ private static Optional<String> makeUseRepoCommand(
       commandParts.add(extensionName);
     }
     commandParts.addAll(repos);
-    return Optional.of(
-        String.format("buildozer '%s' //MODULE.bazel:all", String.join(" ", commandParts)));
+    return Optional.of(String.format("'%s'", String.join(" ", commandParts)));
```
Comment (Wyverald) [inline src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionMetadata.java:357]: nit -- maybe reuse the "Collectors.joining" idiom above?

**Review by Wyverald — APPROVED** [review]

File: src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionMetadata.java (hunk 1 above)
Comment (fmeum) [reply src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionMetadata.java]: Done

**Comment by fmeum** [comment]
@bazel-io flag

**Comment by Wyverald** [comment]
@bazel-io fork 6.4.0

**Comment by meteorcloudy** [comment]
It's probably worth adding a section about managing `use_repo` with buildifier at https://bazel.build/external/migration#best-practices

**Comment by keertk** [comment]
The changes in this PR have been included in [Bazel 6.4.0 RC1](https://releases.bazel.build/6.4.0/rc1/index.html). Please test out the release candidate and report any issues as soon as possible. If you're using Bazelisk, you can point to the latest RC by setting `USE_BAZEL_VERSION=last_rc`.
Thanks!

**Pull request closed without merging**